packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/config_with_when.yaml

```diff
@@ -16,6 +21,10 @@ use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigura
 return static function (ContainerConfigurator $containerConfigurator): void {
     $containerConfigurator->extension('framework', ['test' => false]);
     if ('test' === $containerConfigurator->env()) {
+        $containerConfigurator->extension('not_framework', ['other_test' => true]);
         $containerConfigurator->extension('framework', ['test' => true, 'session' => ['storage_factory_id' => 'session.storage.factory.mock_file']]);
     }
+    if ('prod' === $containerConfigurator->env()) {
+        $containerConfigurator->extension('oky', ['dokky' => false]);
+    }
 };
```

Comment:
Could you add a new separated test fixture, so we maintain the history and reason for the original one?
It is very useful in the hindsight to have test fixture independent on each other.